static/app/components/events/interfaces/frame/category.tsx

```diff
@@ -0,0 +1,51 @@
+import {withTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import RepoLabel from 'app/components/repoLabel';
+import Tooltip from 'app/components/tooltip';
+import {t} from 'app/locale';
+import {FrameCategory} from 'app/types';
+import {Theme} from 'app/utils/theme';
+
+type Props = {
+  theme: Theme;
+  category: FrameCategory;
+};
+
+function Category({category, theme}: Props) {
```

Comment:
as per other comment, I would definetly not call this category. call it `flag`, `label` or anything else... frames can have multiple labels/flags, but the category is something like "ui"/"driver"/etc, if we ever expose this via frontend we will have naming collisions